src/components/Toast.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
```

Comment:
Did we mention this on checkin, is this the way we want to integrate and test our components?

Reply:
I think so, we said that we would create separate files for each component and then import them in the main one.

Reply:
Ok, so for every Jira ticket: [Component]: manual testing. We do this?

Reply:
Yes, and test the events and the stuff that are mentioned in the ticket.

Reply:
Ok gotcha!